.changeset/terrific-apples-run.md

```diff
@@ -0,0 +1,5 @@
+---
+"@fluidframework/tree": minor
+---
+
+Adds garbage collection for repair data so that removed content is not retained forever. This is mostly an internal change so users of SharedTree won't need to adapt any code but it could cause an issue with cross-version collab where an older version does not send a refresher that a newer version may need. In this case, a "refresher data not found" error will be thrown.
```

Comment:
```suggestion
tree: Delete removed content stored in SharedTree

SharedTree now supports garbage collection so that removed content is not retained forever. This is an internal change and users of SharedTree won't need to adapt any existing code.

This change could cause errors with cross-version collaboration where an older client does not send data that a newer version may need. In this case, a "refresher data not found" error will be thrown.
```

@jenn-le I removed some of the Tree-specific terms assuming that those are internal and that customers won't necessarily know what they are. But if I'm wrong about that let me know!

If a customer encounters the "refresher data not found" error what should they do? File an issue? (That's a totally appropriate answer.) I expect that the only customers who would hit this would be internal or very early adopters, since 2.0 itself is only a few weeks old. But the BREAKING CHANGE heading in your PR description is pretty scary since this is a minor release.

Reply:
Ah good call on the tree-specific terms. I had it set to auto merge so if you could get the suggested change in the pr you mentioned, I'd appreciate it. 

If they encounter the error, they could file an issue but should generally make sure that the clients are updated. Cross-version collab issues aren't unique to this change and I'm not sure if we have any other guidance around them.